Sources/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsHandler.swift

```diff
@@ -27,6 +27,29 @@ internal class UIKitRUMViewsHandler: UIKitRUMViewsHandlerType {
         self.predicate = predicate
         self.dateProvider = dateProvider
         self.inspector = inspector
+
+        let nc = NotificationCenter.default
+        nc.addObserver(self, selector: #selector(appWillResignActive), name: UIApplication.willResignActiveNotification, object: nil)
+        nc.addObserver(self, selector: #selector(appDidBecomeActive), name: UIApplication.didBecomeActiveNotification, object: nil)
+    }
+
+    private weak var lastActiveAppVC: UIViewController?
+    @objc
+    private func appWillResignActive() {
+        if lastActiveAppVC != nil {
+            return
+        }
```

Comment:
What if the `lastActiveAppVC` is not `nil` but doesn't match the `lastStartedViewController` ? When the app comes back in background, won't you be starting the wrong RUM View ?

Reply:
Also, I'm wondering if we need the `lastActiveAppVC` property at all. As far as I can see, we either set it to `lastStartedViewController` or to `nil`, so can't we simply do:
* if the app goes to background: stop `lastStartedViewController`, 
* when it goes to foreground: start `lastStartedViewController`
?

If there's sth special about `lastActiveAppVC` that I miss, I think it's worth adding it in the property comment.

Reply:
@xgouchet good point. 
now i remove `lastActiveAppVC` along with @ncreated comments below. it was introduced to guard against multiple notifications received but probably not really needed.